R/install.R

```diff
@@ -33,16 +33,16 @@ install_config <- list(
     "10.2" = list(
       "linux" = list(
         "libtorch" = list(
+          path = "libtorch/",
```

Comment:
Since `path` is constant, it's cleaner to remove it from all lists. This requires updating `source_path` in `lantern_install_libs()` and `lantern_install_lib()`.

Reply:
True! I'll fix this

Reply:
Turns out this is implicitly an empty string for lantern configurations, so I had to revert that change.